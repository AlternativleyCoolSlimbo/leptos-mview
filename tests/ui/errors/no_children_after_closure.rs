use leptos::*;
use leptos_mview::mview;

fn main() {
    mview! {
        Await
            future=[async { 1 }]
        |data| "no"
    };
}