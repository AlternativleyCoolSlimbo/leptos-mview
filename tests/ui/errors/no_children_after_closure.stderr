error: expected children block after closure arguments
 --> tests/ui/errors/no_children_after_closure.rs:8:16
  |
8 |         |data| "no"
  |                ^^^^

warning: use of deprecated method `leptos::AwaitPropsBuilder::<T, Fut, FF, VF, V, ((FF,), __blocking, __local, ())>::build`: Missing required field children
 --> tests/ui/errors/no_children_after_closure.rs:6:9
  |
6 |         Await
  |         ^^^^^
  |
  = note: `#[warn(deprecated)]` on by default

error[E0061]: this method takes 1 argument but 0 arguments were supplied
 --> tests/ui/errors/no_children_after_closure.rs:6:9
  |
6 |         Await
  |         ^^^^^ an argument of type `AwaitPropsBuilder_Error_Missing_required_field_children` is missing
  |
note: method defined here
 --> $CARGO/leptos-0.6.3/src/await_.rs
  |
  | #[component]
  | ^^^^^^^^^^^^
  = note: this error originates in the derive macro `::leptos::typed_builder_macro::TypedBuilder` (in Nightly builds, run with -Z macro-backtrace for more info)
help: provide the argument
  |
6 |         Await(/* AwaitPropsBuilder_Error_Missing_required_field_children */)
  |              +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
